### ISSUE TITLE
[I18N] German and English internationalisation

### ISSUE THREAD

**Issue by Zitrone44** [thm-mni-ii/feedbacksystem#1447]
**Is your feature request related to a problem? Please describe.**
The Feedbacksystem is currently only available in german. To increase the accessibility for international students internationalization (i18n) measures should be implemented to allow usage of the Feedbacksystem in other languages.

**Describe the solution you'd like**
A localisation system should be implemented.
